scrapers/koji.py

```diff
@@ -224,45 +205,6 @@ def update_neo4j(self, builds):
                         commit_hash = child.text.rsplit('#', 1)[1]
                         break
 
-            if not task_dict:
```

Comment:
I am having doubts that line #198 will throw an error if an empty list is returned from teiid. But maybe it will never return an empty list because we are extracting the task_id from the build and the task will always be present. Should we make that statement fail safe? What are your thoughts @mprahl and @sarah256 ?

Reply:
That's a good point. This if statement should be moved to right after `task_dict = self.get_task(task_id)[0]`.